minigalaxy/api.py

```diff
@@ -132,7 +132,8 @@ def get_info(self, game: Game) -> dict:
         return response
 
     # This returns a unique download url and a link to the checksum of the download
-    def get_download_info(self, game: Game, operating_system="linux", dlc_installers="") -> tuple:
+    def get_download_info(self, game: Game, dlc_installers="") -> tuple:
```

Comment:
line 148 is:
```
return self.get_download_info(game, "windows")
```
This will break compatibility with that line.

Reply:
I tried all my Linux games (~ 7) and i never had an problem. Maybe we can just delete "windows" in the return function.